onboarding.md

```diff
@@ -1,49 +1,13 @@
 # Onboarding and first steps
```

Comment:
I would "inline" this onboarding markdown into the README, as currently it's just 13 rows.
Putting it on the main `docs` repository could also make sense, as these first steps shouldn't just be about Connector but EDC in general

Reply:
added the onboarding content to the README. And good point regarding the docs repo, I think that links to the samples anyway in the hands-on section. I'll open a PR there